unstructured/ingest/cli/cmds/azure_cognitive_search.py

```diff
@@ -67,7 +67,13 @@ def azure_cognitive_search_dest(ctx: click.Context, **options):
     log_options(parent_options, verbose=verbose)
     log_options(options, verbose=verbose)
     try:
-        configs = extract_configs(options, validate=[AzureCognitiveSearchCliWriteConfig])
+        runner_cls = runner_map[source_cmd]
+        configs = extract_configs(
+            parent_options,
+            extras={"fsspec_config": FsspecConfig}
+            if issubclass(runner_cls, FsspecBaseRunner)
```

Comment:
That's because this is for the source connector. I recently refactored to introduce this new config, and so each destination connector needs some overhead code to make sure the source connector still runs as expected first.

Reply:
I extracted this logic into its own method so there's less repeat code.

Reply:
ahh okay, got it. yea, extracting makes this cleaner.